shippable.yml

```diff
@@ -24,6 +24,11 @@ script:
   # mubench.reviewsite
   - cd mubench.reviewsite
   - ./configure.sh
+  - java -jar /usr/local/bin/selenium-server-standalone-3.12.0.jar &
+  - cp settings.default.php settings.php
+  - touch test.sqlite
+  - php -S localhost:8080 -t ./ &
+  - sleep 10
   - phpunit --bootstrap vendor/autoload.php tests/
```

Comment:
Separate selenium tests from unit tests, group them in two [test suites](https://stackoverflow.com/questions/38588029/excluding-certain-tests-from-loading-in-phpunit), and execute both/the full suite on shippable.